### DISCUSSION TITLE
First delivery to NISAR ADT on Raider

### DISCUSSION THREAD

**Discussion by dbekaert** [dbekaert/RAiDER#397] [General]
A first delivery to the NISAR ADT from raider is to include the following:
- the software itself. I suggest a released version on github for clear traceability.
- Installation instruction
- A user guide to run the software
- Test data and golden output data to make sure that we can reproduce the expected results; for a cube output with zenith or ray-tracing options using orbit.

Lets us track it here in the discussion on progress.

**Comment by piyushrpt** [comment]
Note that you only need the incidence angle cube for converting Zenith to LOS by projection, and you would need both LOS angles for ray tracing.

**Comment by dbekaert** [comment]
# NISAR ADT RAiDER release update
​
### RAiDER release
v0.2.0
​
### Installing 
To get started, run the following lines in a terminal:
```
conda create --name RAiDER  -c conda-forge raider
conda activate RAiDER
```
Then
```
raider.py --help
```
​
### Supported workflows: 
1. Zenith delays (ZTD) for:
  - the native weather model grid (depends on choice of weather model)
  - a user-defined cube (bounding box, optional: height levels, grid spacing, projection)
​
2. Slant delays (STD) using full raytracing for: 
  - a user-defined cube + orbit file (Sentinel-1 format currently supported)
​
### Supported Weather Models
- HRRR (as used in the config file)
- ECMWF - HRES works with an account
​
### Links to example config files and corresponding output
[Zenith delays on weather model grid nodes at specified height levels](https://github.com/dbekaert/RAiDER/blob/662412cae19dca7c5bea70b0cba1743e471db168/test/scenario_1/raider_example_1.yaml) --> [Output](https://github.com/dbekaert/RAiDER/blob/662412cae19dca7c5bea70b0cba1743e471db168/test/scenario_1/HRRR_tropo_20200101T120000_ztd.nc)   
​
[Slant delays using raytracing on a custom cube](https://github.com/dbekaert/RAiDER/blob/dev/test/scenario_3/raider_example_3.yaml), [(Orbit file)](https://github.com/dbekaert/RAiDER/blob/dev/test/scenario_3/S1A_OPER_AUX_POEORB_OPOD_20181203T120749_V20181112T225942_20181114T005942.EOF) --> [Output](https://github.com/dbekaert/RAiDER/blob/662412cae19dca7c5bea70b0cba1743e471db168/test/scenario_3/HRRR_tropo_20181113T230000_std.nc)
​
​
### Not yet fully supported: 
- Slant delay (STD) calculation by projecting ZTD
- Python library imports
- Slant delay (STD) with line-of-sight file (cube)
